expyfun/_experiment_controller.py

```diff
@@ -940,6 +944,8 @@ def get_presses(self, live_keys=None, timestamp=True, relative_to=None):
             If timestamp==False, returns a list of strings indicating which
             keys were pressed. Otherwise, returns a list of tuples
             (str, float) of keys and their timestamps.
+            If releases is True, the last value in the tuple will be a string,
+            either 'press' or 'release'.
```

Comment:
@Eric89GXL and I also agreed that it should always return a tuple (length 2 if `timestamp=False`, length 3 otherwise) and the last element of the tuple should be `'press'` or `'release'`.